### PR TITLE
Add "Send Feedback…" menu item

### PR DESCRIPTION
**Pull request by karaggeorge**
Adds a `beta.md` template that @skllcrn will edit and a `Send us Feedback` option in the menu that will open a new issue using that template.

screenshot 2018-09-22 10 19 47

### CODE REVIEW

File: main/menus.js [hunk 1]
```diff
@@ -33,6 +33,10 @@ const cogMenu = [
     click: openPrefsWindow
   },
   checkForUpdatesItem,
+  {
+    label: 'Send us feedback…',
+    click: () => shell.openExternal('https://github.com/wulkano/kap/issues/new?template=beta.md')
```
[the remark this answers is not in the document]
Comment (albinekb) [reply main/menus.js]: Doesn't look like you can combine both ?body=<content> and template :/

[the remark this answers is not in the document]
Comment (sindresorhus) [reply main/menus.js]: Ok. I just tested with https://github.com/babel/babel/issues/new?template=Bug_report.md&body=foo So it seems `body` overrides the template. I would honestly prefer to just embed the text in Kap and not use the template feature. That way the Kap version and macOS version are auto-filled.

[the remark this answers is not in the document]
Comment (karaggeorge) [reply main/menus.js]: I tried with the body param, but I think the template is too big and it loses formatting. Unless I'm doing something wrong with it 🤔 

screenshot 2018-09-23 18 10 09

[the remark this answers is not in the document]
Comment (sindresorhus) [reply main/menus.js]: Are you using `encodeURIComponent`? Example here: https://github.com/sindresorhus/caprine/blob/04b8e329d7dec308ea9f934f25d20c7fd521b0ba/menu.js#L112-L123

**Comment by karaggeorge** [comment]
@sindresorhus 

Worked perfectly 👍

File: main/menus.js [hunk 2]
```diff
@@ -20,6 +20,32 @@ const checkForUpdatesItem = {
   }
 };
 
+const issueBody = `
+<!--
+Thank you for helping us test the Kap Beta, your feedback helps us make Kap better for everyone!
```
Comment (sindresorhus) [inline main/menus.js]: I think we should still have the feedback button even after the beta, so let's just use `Kap` and not `Kap Beta`. Alternatively, add a conditional.

**Pull request merged**